### DISCUSSION TITLE
Python Nordeste precisa de patrocínio

### DISCUSSION THREAD

**Discussion by cecivieira** [apyb/comunidade#26] [Geral]
A Python Nordeste, até agora, só conseguiu 1.7k de patrocínio e o evento já vai ser em agosto (faltam 2 meses e meio).

Isso não paga nem a metade do local do evento.

Você trabalha em alguma empresa que patrocina evento? Ou tem contato de alguma? Comenta aqui nessa discussão ou me chama no Telegram (cecivieira).

As cotas de patrocínio são de 15k, 10k, 7k e 4k.

**Comment by ramalho** [comment]
Olá @cecivieira, vou falar com o pessoal de marketing da Thoughtworks. Preciso de mais informações para passar para eles. Onde encontro?

**Reply by cecivieira** [reply]
Obrigada, @ramalho!!

O plano de patrocínio está disponível para download aqui:

https://2022.pythonnordeste.org/#patrocinio

**Reply by rodrigoamaral** [reply]
Muito obrigado pela força, @ramalho !

**Comment by lucianoratamero** [comment]
vou ver aqui na Labcodes também, pra ver se já tão ligados :]

**Reply by cecivieira** [reply]
@lucianoratameto Massa!!! 🧡

**Reply by tupy** [reply]
Estou começando numa startup que tá usando Python, vou tentar conseguir algo também.

**Reply by rodrigoamaral** [reply]
Boa, grande @tupy ! Muito obrigado!

**Reply by rodrigoamaral** [reply]
Obrigado, @lucianoratamero ! O pessoal da Labcodes já está ciente e está analisando. Se você puder dar um reforço, agradecemos bastante.

**Reply by tiidadavena** [reply]
@lucianoratamero @cecivieira falei com o pessoal da Labcodes esta semana e fui informada que eles têm interesse mas que estão analisando outros pontos relacionados à logística do evento.

**Comment by tiidadavena** [comment]
Outras potenciais empresas patrocinadoras seriam a Vinta, Avanade, Instiuto Atlântico. Acho que poderia rolar também uma parceria com a Secretaria Est e/ou Mun. de Educação.

**Reply by gtcarvalh0** [reply]
Oi, já levei pra Vinta e eles já estão analisando :)

**Reply by rodrigoamaral** [reply]
Joia, @gtsalles ! Obrigado pela força!
